profiler/src/ProfilerEngine/Datadog.Profiler.Native/StackFramesCollectorBase.cpp

```diff
@@ -119,9 +86,9 @@ StackSnapshotResultBuffer* StackFramesCollectorBase::CollectStackSampleImplement
     // The actual business logic provided by a subclass goes into the XxxImplementation(..) methods.
     // This is a fallback implementation, so that the implementing sub-class does not need to overwrite this method if it is a no-op.
 
-    bool frame1Added = TryAddFrame(StackFrameCodeKind::Dummy, 0, 0, 0);
-    bool frame2Added = TryAddFrame(StackFrameCodeKind::Dummy, 0, 0, 0);
-    bool frame3Added = TryAddFrame(StackFrameCodeKind::Dummy, 0, 0, 0);
+    bool frame1Added = AddFrame(1);
```

Comment:
Maybe just 1 AddFakeFrame() instead of this weird list of 3 "dummy" frames with 1, 2 and 3 as instruction pointers